src/shogun/machine/LinearMachine.cpp

```diff
@@ -120,3 +141,50 @@ void CLinearMachine::store_model_features()
 {
 }
 
+float64_t CLinearMachine::compute_bias(CFeatures* data)
+{
+	REQUIRE(m_labels,"No labels set\n");
+
+	if (!data)
+		data=features;
+
+	REQUIRE(data,"No features set\n");
```

Comment:
Can you be more informative here: No features provided and no featured previously set